docs/concepts/intro.rst

```diff
@@ -0,0 +1,41 @@
+.. meta::
+  :description: rocPRIM documentation and API reference library
+  :keywords: rocPRIM, ROCm, API, documentation
+
+.. _rocprim-intro:
+
+********************************************************************
+ Introduction to rocPRIM
+********************************************************************
+
+
+Operations and Sequences
+========================
+
+A rocPRIM operation is a computation over a sequence of objects returning one value like ``reduce``, returning another sequence like ``sort``, or multiple sequences like ``partition``. The elements of the sequence could be of any type or class, although template specialization allows rocPRIM to optimize the computations over the usual numerical datatypes. Operations handle sequences by expecting ``iterators`` as input and mutable ones as output.
```

Comment:
This is just a suggestion about something that's not a huge deal, but I noticed there is a little ambiguity in the wording here in a few places. I have two small suggestions that I thought might help make it clearer, but feel free to disregard this if you think it's not worth it.

1. In the first sentence, to eliminate any ambiguity about whether "one value like reduce" means reduce is a value or an operation: "A rocPRIM operation is a computation over a sequence of objects. An operation's output may be a single value (eg. the ``reduce`` operation), another sequence (eg. the ``sort`` operation), or multiple sequences (eg. the ``partition`` operation)."
2. In the last sentence, to remove ambiguity about what "ones" refers to: "Operations accept input in the form of ``iterators`` that point to a sequence of objects to process. They write their output to a mutable ``iterator``.

Reply:
Fixed as suggested.